### PR TITLE
Update Readme.md for Content app v2.0.0

### PR DESCRIPTION
**Pull request by Mihaiii**
Update Readme.md for Content app v2.0.0.

All the links for Website's readme.md are for `develop` branch. I will update the file once Content app's `develop` merges into `master`.

### CODE REVIEW

**Comment by Mihaiii** [comment]
@Starcounter/sc-apps , please review.

**Comment by Mihaiii** [comment]
@Mackiovello Could you please have a look at this PR? :)

File: README.md [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
-It can be found at `/content/cms`.
+## Requirements
+
+This app requires [SignIn app](https://github.com/starcounterapps/signin) to run simultaneously to work properly. It will only display the user interface for users that are signed in. See the [instructions for the SignIn app](https://github.com/StarcounterApps/SignIn#create-the-first-user) to create a first user. 
+
+## Description
+
+### Main view
+
+The main view (which provides the possibility to create, view, delete, update) consists in a list of content blocks.
```
Comment (Mackiovello) [inline README.md]: I would rephrase to something like this:

> The main view is a list of the content blocks and buttons to create, view, delete, and update them.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
+A content item has the following properties: `Name`, `Url` (it is autogenerated), `Parent Page`, `Visibility` (or `Protected`), `Icon` and `Html` (the last one in not shown in the main view).
 The `Url` property specifies how we can access the content item.
-If the `Protected` property is set to `True`, the content will only be available for authenticated users, otherwise, an empty response will be returned.
+If the `Visibility` property is set to `Only logged in users`, the content will only be available for authenticated users, otherwise, an empty response will be returned.
+`Parent page` and `Icons` are used for in the "Table of Contents" (`http://localhost:8080/content/toc`).
 The `Html` property stores the HTML of the item.
 
+### Table of Contents
+
+The "Table of Contents" page allows the user to view all content items names within a tree menu. When the name of an item is clicked, its content is displayed on the right side of the page.
```
Comment (Mackiovello) [inline README.md]: within -> in

allows the user to view -> lets the user view

File: README.md [hunk 3]
```diff
@@ -39,30 +55,41 @@ All modes can be previewed in the "Preview" tab.
...
+We go to `http://localhost:8080/websiteeditor` and add a new [Surface](https://github.com/StarcounterApps/Website/tree/develop#surfaces) by pressing on the "+" icon (next to the default "Unassigned" group) and selecting one. For this example, we will use "Sidebar":
 
-![](/images/surfaces.png)
+![](/images/website_surface.png)
 
-The surface has two [blending points](https://github.com/StarcounterApps/Website#blending-points): "Left" and "Right". We want our content item to appear in the left slot and also want that the main page of "DesktopSurface" to appear in the right slot. For this we add the two blending points and make the right one default (please notice that the "Main" blending point of the "DesktopSurface" is also default).
+We got to the newly created surface (the default name is "New Sidebar surface") and on [Routes](https://github.com/StarcounterApps/website/tree/develop#routes) we add the URI where we want to insert the Content item that we just created. Fot this example, we'll use `/WebsiteEditor`.
```
Comment (Mackiovello) [inline README.md]: we got to the -> we go to the

Fot this example -> For this example

File: README.md [hunk 4]
```diff
@@ -39,30 +55,41 @@ All modes can be previewed in the "Preview" tab.
...
+2. Navigate to `http://localhost:8080/search/settings`
+3. Sign in if you aren't already
+4. Remove all Concepts and add "Simplified.Ring6.WebContent" so that the search will be made only on this table (Content app's database table).
 
-## Requirements
+![](/images/search_app_settings.png)
+
+5. Restart Search app (not needed if at step 4 "Simplified.Ring6.WebContent" was already added)
+6. Navigate to Content app's "Table of Contents": `http://localhost:8080/content/toc`.
+7. The search will be made when "GO!" button is pressed.
```
Comment (Mackiovello) [inline README.md]: when "GO!" button -> when the "GO!" button

**Comment by Mihaiii** [comment]
Thank you very much for the review!
I made the requested changes and now I'm merging.

> Dang, the content app is starting to look really good!

The next version ( #110 ) will look fabulous :)
